butterfly-test/src/main/resources/log4j2.xml

```diff
@@ -3,7 +3,7 @@
 	<Appenders>
 		<Console name="CONSOLE" target="SYSTEM_OUT" follow="true">
 			<PatternLayout>
-				<Pattern>[%d{HH:mm:ss.SSS}] [%highlight(%level)] %msg%n</Pattern>
+				<Pattern>[%d{HH:mm:ss.SSS}] [%highlight{%level}] %msg%n</Pattern>
```

Comment:
Why is this change necessary? This works as is.

Reply:
But indeed, that link show the usage of curly braces instead of parenthesis. Did you test this change? Did the loglevel get colored?

Reply:
Cool, thanks

Reply:
Strange though that highlights worked for you. hmm